pocs/utils/pyro/pyro_camera_server.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
Please add file level comments describing the purpose.
And should this be in POCS/scripts? That is a true question, not a statement at all.

Reply:
Agree on the comments, unsure on file location. It is a script, but also an importable function, and could be used either way. It's in utils right now mainly because I'm trying to get coverage to include it (scripts aren't included, but could be with a change to coveragerc).

Reply:
Comment added.

Reply:
I do the same thing for `pocs/utils/data.py` but want to move away from it. I think we just leave everything within `pocs/` as importable modules and just create a thin-wrapper in `scripts`.

Reply:
OK, that makes sense. I'll keep the functions that do all the work in `$POCS/pocs/utils/pyro` and have minimal scripts in `$POCS/scripts` that just parse the command line argument and call the function.